src/Compilers/CSharp/Portable/Emitter/Model/AttributeDataAdapter.cs

```diff
@@ -173,6 +173,22 @@ private Cci.IMetadataNamedArgument CreateMetadataNamedArgument(string name, Type
             return new MetadataNamedArgument(symbol, moduleBeingBuilt.Translate(type, syntaxNodeOpt: (CSharpSyntaxNode)context.SyntaxNodeOpt, diagnostics: context.Diagnostics), value);
         }
 
+        private protected sealed override bool IsStringProperty(string memberName)
```

Comment:
>private protected sealed override bool IsStringProperty(string memberName) [](start = 8, length = 74)

I think this code belongs in src\Compilers\CSharp\Portable\Symbols\Attributes\AttributeData.cs. It is not emit specific, quite the opposite.